### DISCUSSION TITLE
Lines of Code for Implementation

### DISCUSSION THREAD

**Discussion by twharmon** [krausest/js-framework-benchmark#1527] [Ideas]
Would people benefit from also seeing how many lines of code were used in the implementations?

I'm not sure if there is a reliable way to report how much code is required. Might have to run Prettier or something first to prevent implementations from cheating.

**Comment by leeoniya** [comment]
i can see this leading to an arms race of code-golfing that sacrifices legibility for an LoC metric. i guess if @krausest curates each submission to prevent this, then maybe it can be informative. most implementations are pretty close from what i've seen, but there could be some outliers. not sure how this would work for WASM frameworks, where the source isn't JS, and could contain various amounts of standard boilerplate or compiler config.

**Comment by fabiospampinato** [comment]
Probably best to not show this metric imo, the thing that matters to the user is if the implementation looks reasonable (clean, maintainable, no/few tricks etc.), which can't be really quantified into a number, interested users should just look at the code directly.
